docs/README.md

```diff
@@ -0,0 +1,107 @@
+# Octokit Requirements
+
+> Provide a first class experience for platform developers in their preferred programming languages
+
+Octokit strives to offer complete and officially supported SDKs, quick starts, code samples, and copy/paste-able examples for the GitHub Platform in the most popular languages.
+
+Each Octokit will work towards meeting these requirements:
+
+|                                 | JavaScript | Ruby | .NET |
+| ------------------------------- | ---------- | ---- | ---- |
+| [Complete](#complete)           | 🌔         | 🌔   | 🌔   |
+| → REST                          | ✅         | ✅   | ✅   |
+| → GraphQL                       | 🌑         | 🌑   | 🌑   |
+| → GitHub Apps                   | 🌑         | 🌑   | 🌑   |
+| → OAuth                         | 🌑         | 🌑   | 🌑   |
+| → Webhooks                      | 🌑         | 🌑   | 🌑   |
+| [Decomposable](#decomposable)   | 🌑         | 🌑   | 🌑   |
+| [Generated](#generated)         | 🌑         | 🌑   | 🌑   |
+| [Native](#native)               | 🌑         | 🌑   | 🌑   |
+| [Documented](#documented)       | 🌑         | 🌑   | 🌑   |
+| [Tested](#tested)               | 🌑         | 🌑   | 🌑   |
+| [Friendly](#friendly)           | 🌑         | 🌑   | 🌑   |
+| [Prescriptive](#prescriptive)   | 🌑         | 🌑   | 🌑   |
+| [Supported](#supported)         | 🌑         | 🌑   | 🌑   |
+| [Maintained](#maintained)       | 🌑         | 🌑   | 🌑   |
+| [Collaborative](#collaborative) | 🌑         | 🌑   | 🌑   |
+
+Key:
+
+🌑 Not started — 🌔 In progress —  ✅ Meets requirements
+
+## Complete
+
+The SDK includes support for all platform features:
+
+- REST - by offering a generic HTTP request library and a wrapper method for each endpoint
+- GraphQL - by offering a generic GraphQL client that is pre-configured and optimized for GitHub
+- GitHub Apps - by implementing all REST endpoints for GitHub Apps as well as JWT authentication
+- OAuth - by offering middleware in the most popular web framework for performing the OAuth exchange
+- Webhooks - by offering middleware in the most popular web framework for validating webhooks
+
+## Decomposable
+
+Each feature is available both as individual packages and one monolithic meta-package that includes all components pre-wired.
+
+## Generated
+
+Any part of the SDK that varies based on the target GitHub instance should be generated from a schema.
```

Comment:
Maybe ... from "the common schema". Just so we don't end up with multiple schemas :)